README.md

```diff
@@ -1,4 +1,4 @@
-# OpenDocument.ios
+# OpenDocument.ios ![](https://github.com/marcelblanarik/OpenDocument.ios/workflows/build/badge.svg)
```

Comment:
please change link to original repository so it immediately works after merge

Reply:
done